src/svc-cloud-data-sources.yml

```diff
@@ -0,0 +1,24 @@
+openapi: 3.1.0
```

Comment:
the swagger parser we are using only supports up to 3.0.3, is there a specific 3.1.0 swagger feature you're using?

Reply:
I have no idea. :D